src/proto/api_messages.js

```diff
@@ -225,7 +226,7 @@ class AccountCreationRequest {
    */
   toArray(includeLabel = true) {
     const arr = [
-        this.complete_, // field 1 - complete
```

Comment:
I think these formatting changes can be backed out since this file is just autogenerated and copy-pasted over from the generator.

Reply:
Thats my guess. This file shouldn't be modified by an editor since it's only copy-pasted out. You can reset it back to the main line with `git checkout origin/main src/proto/api_messages.js` or `git checkout me/main src/proto/api_messages.js` if you're using the `me/team` remote setup from the docs

Reply:
Oh wow. I don't have time to fix this, but someone should really add a warning to this file with a link to the documentation.